### ISSUE TITLE
Tor browser s profile need an update for using with ubuntu 16.04?

### ISSUE THREAD

**Issue by ccchan234** [netblue30/firejail#1959]
hi~

I am on ubuntu 16.04 + latest? firejail + lastest tor browser.

I googled and found many people refer to this profile for tor 's profile:
https://github.com/netblue30/firejail/blob/master/etc/start-tor-browser.profile

but when I run with firejail,
it say cant find some files.

say from the above profile:
```
# Firejail profile for start-tor-browser
# This file is overwritten after every install/update
# Persistent local customizations
include /etc/firejail/start-tor-browser.local
# Persistent global definitions
include /etc/firejail/globals.local


include /etc/firejail/disable-common.inc
include /etc/firejail/disable-devel.inc
include /etc/firejail/disable-interpreters.inc
include /etc/firejail/disable-passwdmgr.inc
include /etc/firejail/disable-programs.inc
```

the 4th line "/etc/firejail/start-tor-browser.local" this file is NOT found in my /etc/firejail/
I commented it out,
then it say "/etc/firejail/disable-interpreters.inc" this file not found.
"Error: cannot open profile file /etc/firejail/disable-interpreters.inc"

would be kind if someone use ubuntu 16.04, apt-get the firejail,
dnld latest tor browser could point out how to solve it.

currently, I do can run tor inside firejail by
firejail --noprofile ./start-tor-browser inside tor's folder.

And... may I ask without profile, will the above run be "safe" or at least with some help?

thank you.

**Comment by Fred-Barclay** [comment]
@ccchan234 can you post the output of `firejail --version`? IIRC the latest version available in Ubuntu 16.04's repositories is 0.9.38. We didn't add etc/firejail/disable-interpreters.inc until our last release, 0.9.54, so you'll need to comment out that option (and possibly others) if you want to use this profile with the firejail version in your repositories.

> And... may I ask without profile, will the above run be "safe" or at least with some help?

It's no less "safe" than running tor browser without firejail, but it doesn't offer much protection.

**Comment by ccchan234** [comment]
hi~
my version is firejail version 0.9.38.10

btw, i am a layman so I used firejail for long time but didn't edit the profiles before.
For newbies like me who may only want to use the default profile of FJ 0.9.38.10 with tor,
what I have to do is cd into tor's folder that contain the "start-tor-browser" script,
typically "tor-browser_en-US/Browser",
and run as "firejail --private=/home/user/xxxxx ./start-tor-browser".
the --/home/user/xxxx is the folder where the root of the "firejail-ed tor-browser" will be.

I think the default profile will give SOME protection while their customised profile for tor
will give MORE protection?
https://github.com/netblue30/firejail/blob/master/etc/start-tor-browser.profile
